packages/socket-mode/src/SocketModeClient.ts

```diff
@@ -398,6 +398,37 @@ export class SocketModeClient extends EventEmitter {
       }
     });
   }
+
+  /**
+   * Removes secrets and tokens from socket request and response objects
+   * before logging.
+   * @param body - the object with values for redaction.
+   * @returns the same object with redacted values.
+   */
+  private static redact(body?: Record<string, unknown>): Record<string, unknown> | unknown[] | undefined {
+    if (body === undefined || body === null) {
+      return body;
+    }
+    const records = Object.create(body);
```

Comment:
nit: record? this value should be a singler object

Reply:
Nice catch and thanks for calling it out! Updated 🙏